include/git2/transaction.h

```diff
@@ -8,6 +8,14 @@
 #define INCLUDE_git_transaction_h__
 
 #include "common.h"
+
+/**
+ * @file git2/transaction.h
+ * @brief Git transaction routines.
+ * @defgroup Git transaction routines.
```

Comment:
This should be `@defgroup git_transaction Git transaction routines` instead. Please also remove the trailing dots from `@brief` and `@defgroup`.